.gitignore

```diff
@@ -3,3 +3,4 @@ src/locales
 *.pid
 main.js
 eslint.report.html
+lib
```

Comment:
https://stackoverflow.com/questions/24942161/does-npm-ignore-files-listed-in-gitignore

Just make sure that the published package is actually will contain the lib folder. Last time somebody added lib to gitignore and the resulting package didn't have any code...